### ISSUE TITLE
Configuration for LDAP

### ISSUE THREAD

**Issue by stboissdev** [stevespringett/Alpine#13]
I am not arrived to configure LDAP in the application.properties file. 
however, I can do a search via ldapsearch
```
ldapsearch -l 0 -d 0 -vvvv -H ldap://myldap:389/ -b "ou=people,dc=domain,dc=com" -D "uid=myaccess,ou=functionalaccounts,dc=domain,dc=com" -W  "uid=myid"
```

Can you help me ?

**Comment by stevespringett** [comment]
Based on the above search example, the configuration would likely look something like this:

```ini
alpine.ldap.enabled=true
alpine.ldap.server.url=ldap://myldap:389
alpine.ldap.domain=domain.com
alpine.ldap.basedn=dc=domain,dc=com
alpine.ldap.bind.username=myserviceAccountUsername
alpine.ldap.bind.password=myServiceAccountPassword
alpine.ldap.attribute.name=userPrincipalName
alpine.ldap.attribute.mail=mail
```

**Comment by stevespringett** [comment]
If you're still having issues with LDAP integration, PR #14 and #15 adds additional configuration parameters which should be useful. These will be released with Alpine 1.2.0 included in Dependency-Track v3.1.0.

**Comment by stevespringett** [comment]
Closing. If you're still having issues, please reopen.
